features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Base/Header/avatar/Avatar.jsx

```diff
@@ -67,6 +67,11 @@ class Avatar extends Component {
                 openMenu: !this.state.openMenu,
                 profileIcon: event.currentTarget,
             });
+        } else {
+            this.setState({
+                openMenu: false,
+                profileIcon: this.state.profileIcon,
```

Comment:
shall we destructure the `profileIcon` as a variable at the top of the component?